src/NuGet.Clients/PackageManagement.VisualStudio/Migrate/ProjectJsonToPackageRefMigrator.cs

```diff
@@ -0,0 +1,146 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Shell.Interop;
+using NuGet.Common;
+using NuGet.Frameworks;
+using NuGet.LibraryModel;
+using NuGet.ProjectModel;
+using Task = System.Threading.Tasks.Task;
+using EnvDTE;
+
+namespace NuGet.PackageManagement.VisualStudio
+{
+    public class ProjectJsonToPackageRefMigrator
```

Comment:
This seems like it should be a static class unless something is actually using this to store state.